bin/console

```diff
@@ -2,7 +2,7 @@
 # frozen_string_literal: true
 
 require 'bundler/setup'
-require 'imagen_rb'
```

Comment:
We should probably remove `bin/console` and `bin/setup`. It's a default added by `bundler gem ...`